app/src/main/java/com/goldenraven/padawanwallet/wallet/wallet/WalletHome.kt

```diff
@@ -73,13 +70,16 @@ class WalletHome : Fragment() {
         viewModel.balance.observe(viewLifecycleOwner, {
             if (viewModel.satoshiUnit.value == true) {
                 val balanceInSatoshis = it
-                val humanReadableBalance = DecimalFormat("###,###,###").format(balanceInSatoshis.toLong()).replace(",", "\u2008")
+                val humanReadableBalance =
```

Comment:
Not sure if this is a style being applied automatically by your IDE or if you did those little fixes manually, but let's keep the formatting refactoring out of the bug fixes PRs, particularly because we haven't really defined a standard way to handle those across the app.